Bottom: 57c4f07f976a75c65a0149722a7b729c249084c2
Top:    d356ad690f6ce6bdad48187578fbf9e4eb92fb29
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 15:41:19 +0100

draw.c: Drop unused area parameters from Redraw(...)


---

diff --git a/src/draw.c b/src/draw.c
index 1415b2e..39605ca 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -80,7 +80,7 @@ static const BoxType *clip_box = NULL;
 /* ---------------------------------------------------------------------------
  * some local prototypes
  */
-static void Redraw (bool, BoxTypePtr);
+static void Redraw (void);
 static void DrawEverything (BoxTypePtr);
 static void DrawPPV (int group, const BoxType *);
 static int DrawLayerGroup (int, const BoxType *);
@@ -202,7 +202,7 @@ Draw (void)
 void
 RedrawOutput (BoxTypePtr area)
 {
-  Redraw (true, area);
+  Redraw ();
 }
 
 /* ---------------------------------------------------------------------------
@@ -218,12 +218,10 @@ ClearAndRedrawOutput (void)
 
 
 /* ---------------------------------------------------------------------- 
- * redraws all the data
- * all necessary sizes are already set by the porthole widget and
- * by the event handlers
+ * redraws all the data by the event handlers
  */
 static void
-Redraw (bool ClearWindow, BoxTypePtr screen_area)
+Redraw (void)
 {
   gui->invalidate_all ();
 }
